internal/pkg/cli/deploy/env.go

```diff
@@ -133,6 +172,7 @@ func (d *envDeployer) DeployEnvironment(in *DeployEnvironmentInput) error {
 		CustomResourcesURLs:  in.CustomResourcesURLs,
 		ArtifactBucketARN:    s3.FormatARN(partition.ID(), resources.S3Bucket),
 		ArtifactBucketKeyARN: resources.KMSKeyARN,
+		PrefixListIDs:        prefixListIDs,
```

Comment:
Reading the field name it's not clear to me what the prefixes are _for_? What do you think of renaming to `EC2PrefixListIDs` or `CIDRPrefixListIDs` (same for the field name in `template`)